autoconfigure-storage/src/main/java/com/google/cloud/trace/zipkin/autoconfigure/ZipkinStackdriverStorageProperties.java

```diff
@@ -16,26 +16,36 @@
 
 package com.google.cloud.trace.zipkin.autoconfigure;
 
+import lombok.Data;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
 @ConfigurationProperties("zipkin.storage.stackdriver")
+@Data
 public class ZipkinStackdriverStorageProperties {
   private String projectId;
   private String apiHost = "cloudtrace.googleapis.com";
+  private Executor executor = new Executor();
```

Comment:
How does this work with the configuration yml file? Can the user set zipkin.storage.stackdriver.executor.corePoolSize?